Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -172,20 +170,11 @@ def create_revert_commit_msg(cls, args, commit_objects, **kwargs):
             return None, None
 
         revert_radar_link, revert_bug_link = None, None
-        if isinstance(revert_issue.tracker, radar.Tracker):
-            revert_radar_link = revert_issue.link
-            refs = [i for i in revert_issue.references if isinstance(i.tracker, bugzilla.Tracker)]
-            if refs:
-                revert_bug_link = refs[0].link
-        else:
-            revert_bug_link = revert_issue.link
-            for bug in CommitProgram.bug_urls(revert_issue):
-                for regex in cls.RES:
-                    if regex.match(bug):
-                        revert_radar_link = bug
-                        break
-                if revert_radar_link:
-                    break
+        for issue_url in CommitProgram.bug_urls(revert_issue):
+            if isinstance(Tracker.from_string(issue_url).tracker, radar.Tracker):
+                revert_radar_link = issue_url
+            if isinstance(Tracker.from_string(issue_url).tracker, bugzilla.Tracker):
+                revert_bug_link = issue_url
```

Comment:
This is fine for now, although we can generalize this too later. You aren't making this situation worse, though, so that shouldn't block landing this change.